lib/tokens/token.js

```diff
@@ -41,41 +41,13 @@ module.exports = function (log, config, random, P, hkdf, Bundle, error) {
     this.createdAt = details.createdAt || 0
```

Comment:
Every time I see this line, it confuses me. Can someone remind me why we need `|| 0` again please?